Symfony/CS/Tests/Fixer/Contrib/EmptyPhpdocFixerTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ */
+final class EmptyPhpdocFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        return array(
+            array(
+                '<?php
+                    /** a */
+
+                    '.'
+
+                    '.'
+
+                    '.'
+
+                    '.'
+                    /**
+                     * test
+                     */
+
+                     /** *test* */
+                ',
+                '<?php
+                    /**  *//** a *//**  */
+
+                    /**
+                    */
+
+                    /**
+                     *
+                     */
+
+                    /** ***
```

Comment:
It's a user case thing. Taking things strictly - this is part of docblock, so it's not empty...
I don't like them too, but still...